nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/config/ConfigBeanJMXSupport.java

```diff
@@ -526,6 +527,7 @@ private void findStuff(
                 attrs.add(a);
                 if ( a.returnType() != String.class )
                 {
+                    // FIXME DEADLOCK
```

Comment:
Can you add some more details about the deadlock?

Reply:
Removed